packages/sfc-playground/src/App.vue

```diff
@@ -42,11 +37,11 @@ const sfcOptions = {
   },
   style: {
     isProd: !useDevMode.value
-  },
+  } as SFCAsyncStyleCompileOptions,
   template: {
     isProd: !useDevMode.value
-  }
-}
+  } as SFCTemplateCompileOptions
```

Comment:
I don't know what happened in repl package.

Related: https://github.com/vuejs/repl/issues/83

Reply:
1.3.5 commits have been pushed.